src/templates/about-page.js

```diff
@@ -56,33 +61,52 @@ export const AboutPageTemplate = ({
                 width: '80%',
                 marginLeft: '10%',
               }}>
-                <h2 className="is-size-3">{heading}</h2>
+                <h2 className="is-size-4">{headingAboutTechConnect}</h2>
                 <hr style={{
                   color:'#1C2833',
                   backgroundColor: '#1C2833',
                   height: 5
                 }}/>
-                <p >{description}</p>
+                <Row gutter={40}>
+                  <Col span={12}><p className="is-size-5">{aboutTechConnect}</p></Col>
+                  <Col span={12}><PreviewCompatibleImage imageInfo={image2} /></Col>
+                </Row>
               </div>
+              <div style={{
+                width: '80%',
+                marginLeft: '10%',
+                paddingTop: '5%'
+              }}>
+                <h2 className="is-size-4">{headingAboutMAF}</h2>
+                <hr style={{
+                  color:'#1C2833',
+                  backgroundColor: '#1C2833',
+                  height: 5,
+                }}/>
+                <Row gutter={40}>
+                  <Col span={12}><PreviewCompatibleImage imageInfo={image3} /></Col>
+                  <Col span={12}><p className="is-size-5">{aboutMAF}</p></Col>
+                </Row>
+            </div>
             </div>
             <div className="full-width-image-container"
             style={{
               background:'#1C2833',
               color: 'white',
-              marginTop: '3%',
-            }}><h2 className="is-size-4"
+              marginTop: '1%',
+            }}><h2 className="is-size-3"
             style={{
-              width: '60%',
-              fontFamily: 'calibri',
-              textAlign: 'center'
+              width: '80%',
+              color: 'white',
+              textAlign: 'center',
+              fontStyle: 'italic'
             }}
-            >{quote}</h2></div>
-            <div>
-              <Button size="large" href="/" style={{marginLeft: '17vw'}}>Create a Project</Button>
-              <Button size="large" href="/" style={{marginLeft: '13vw'}}>Join an Existing Project</Button>
-            </div>
+            >{quote}
+            <br /><br /><Button size="large" href="/">Share a Project</Button>
```

Comment:
Can we make this markdown-controlled? Also, I feel like it should be **bold** or ALL CAPS or **BOTH**...

Reply:
I like the bold and markdown-controlled ideas, but making it all caps looked a little aggressive and inconsistent with the rest of the page in my opinion. Here are the two side-by-side... do you have a preference? I also made the quote text bold, looks nice I think.
<img width="1180" alt="Screen Shot 2019-03-18 at 8 38 15 PM" src="https://user-images.githubusercontent.com/35846939/54578678-e1f3ab00-49bd-11e9-85fe-b94b2f5765d8.png">
<img width="1145" alt="Screen Shot 2019-03-18 at 8 38 37 PM" src="https://user-images.githubusercontent.com/35846939/54578680-e28c4180-49bd-11e9-9c1d-e89a04c895be.png">

Reply:
I think I like the italics better than the bold for the quote. I'm fine with either option for the button, especially if it's markdown-controlled so it's easy to change whenever... :-P